src/doc/book/primitive-types.md

```diff
@@ -171,12 +171,10 @@ and a length.
 ## Slicing syntax
 
 You can use a combo of `&` and `[]` to create a slice from various things. The
-`&` indicates that slices are similar to [references], which we will cover in
+`&` indicates that slices are similar to [references][references], which we will cover in
```

Comment:
CommonMark, which pandoc is supposed to implement [explicitly allows this][cm].

[cm]: http://spec.commonmark.org/0.22/#shortcut-reference-link